Versions-ios.plist.in

```diff
@@ -48,6 +48,7 @@
 			<string>14.5</string>
 			<string>15.0</string>
 			<string>15.2</string>
+			<string>15.4</string>
```

Comment:
What happened to 15.3?

Reply:
No idea, what is shipped on Xcode 13.3 is 15.4 I wonder if they are moving to even numbers releases just like us, it seems that there is not 15.1 either heh